packages/cli/src/commands/destroy.js

```diff
@@ -0,0 +1,6 @@
+export const command = 'destroy <type>'
+export const aliases = ['d']
+export const desc = 'Rollback changes made by generate command.'
+
+export const builder = (yargs) =>
+  yargs.commandDir('./destroy', { recurse: true }).demandCommand().argv
```

Comment:
We changed some things in master, `argv` is no longer required.
```suggestion
  yargs.commandDir('./destroy', { recurse: true }).demandCommand()
```